api_app/api/routes/templates.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from fastapi import HTTPException
+from starlette import status
+
+from db.errors import DuplicateEntity, EntityDoesNotExist
+from db.repositories.resource_templates import ResourceTemplateRepository
+from models.domain.resource import ResourceType
+from resources import strings
+
+
+def get_current_template_by_name(template_name: str, template_repo: ResourceTemplateRepository, resource_type: ResourceType, parent_service_template_name: str = "", is_update: bool = False) -> dict:
```

Comment:
This was moved from https://github.com/microsoft/AzureTRE/pull/1457/files#diff-b82ed3b790102de55f2ccb34b09e72cb9dcea8b64418d34f4f8e85463d89e757